src/effects/effectchainslot.cpp

```diff
@@ -192,6 +192,15 @@ void EffectChainSlot::loadEffect(const unsigned int iEffectSlotNumber,
                                                  m_enabledInputChannels);
 }
 
+void EffectChainSlot::refreshAllEffects() {
+    for (EffectSlotPointer pSlot : m_effectSlots) {
+        if (pSlot->isLoaded()) {
+            pSlot->loadEffect(pSlot->getManifest(), pSlot->getInstantiator(), m_enabledInputChannels);
```

Comment:
This makes EffectChainSlot and EffectSlot tightly coupled. Instead of using getter functions to access EffectSlot's private members, make a new EffectSlot::refresh function.